gno.land/pkg/gnoland/app.go

```diff
@@ -430,11 +430,9 @@ func EndBlocker(
 		}
 
 		// Run the VM to get the updates from the chain
-		response, err := vmk.QueryEval(
-			ctx,
-			valRealm,
-			fmt.Sprintf("%s(%d)", valChangesFn, app.LastBlockHeight()),
-		)
+		expr := fmt.Sprintf("%s(%d)", valChangesFn, app.LastBlockHeight())
+		msgEval := vm.NewMsgEval(vm.FormatDefault, valRealm, expr)
```

Comment:
@zivkovicmilos, we don't need to port this module in this PR, but I'd like your opinion on what the diff will look like thanks to the new JSON built-in support.